src/deployment/deploylib/configuration.py

```diff
@@ -107,6 +112,59 @@ def parse_nsg_json(self, config: Any) -> None:
         self.allowed_ips = proxy_config["allowed_ips"]
         self.allowed_service_tags = proxy_config["allowed_service_tags"]
 
+    def parse_endpoint_json(self, config: Any) -> None:
+
+        if "cli_client_id" not in config:
+            raise Exception(
+                "CLI client_id not provided as valid key. Please Provide Valid Config."
+            )
+
+        if (
+            not isinstance(config["cli_client_id"], str)
+            or config["cli_client_id"] == ""
```

Comment:
probably client ID then should be also a valid GUID if you want to do validation here

Reply:
Done.